src/subsplit_dag.hpp

```diff
@@ -30,6 +54,7 @@ class SubsplitDAG {
   SubsplitDAG();
   explicit SubsplitDAG(const RootedTreeCollection &tree_collection);
 
+  // Root node is not counted towards total node count.
```

Comment:
🤔 this is non-obvious. If we want to maintain this behavior, should we call it something else, like `SubsplitCount`? Have a think.